src/app/game-state/main-loop.service.ts

```diff
@@ -100,7 +100,11 @@ export class MainLoopService {
           this.tickCount++;
           if (this.tickCount >= this.tickDivider){
             this.tickCount = 0;
-            this.tick();
+            if (this.pause) {
```

Comment:
I'm medium sure that you can't get to this point in the code if the pause flag is set. Did you test this?

Reply:
Oh, wait. I could be very wrong since the subscribers can set the pause flag.

Reply:
![image](https://user-images.githubusercontent.com/3268868/179375034-f56e8faa-2b8f-44d4-b72f-be9158587d8d.png)

Based on my testing, I believe this does indeed function as intended.

Reply:
Anecdote: I've always had at least 3 days added after death. I'm stopping at 0 every time on this PR, so the fix definitely works.